### ISSUE TITLE
No/broken documentation for API 0.3

### ISSUE THREAD

**Issue by simonpoole** [osm-fr/osmose-frontend#375]
See https://wiki.openstreetmap.org/wiki/Osmose/api/0.3 the documentation link literally points to a 404 page.

**Comment by frodrigo** [comment]
Yes. It is a know issue. There is a (slow) in progress migration and the documentation is no more on line.
But yes, it was a 404 pages listing available URLs, no more.

Most of the URL of the API area available from this page, using the parameters listed on wiki.
http://osmose.openstreetmap.fr/fr/issues/open?item=7170&class=200

If you have questions. I can help you.

**Comment by frodrigo** [comment]
- `auto` is when only one translation is returned based on `accept-language` header or `langs` parameter https://wiki.openstreetmap.org/wiki/Osmose/api/0.3#meta
- Some title may content `{}` from JSOM MapCSS rules. JOSM does not have required concept matching the Osmose title one. So the string with place holder may be used for that. The place holder replaced version is used in subtitle. It is only the case for rule generated from JOSM MapCSS.

Note the preferred way to access the data in 0.3 is using vector tiles and not free bounds json.
```
http://osmose.openstreetmap.fr/api/0.3/issues/{z}/{x}/{y}.mvt?item=7170&level=1,2,3
```
Same content as non full json.

**Comment by simonpoole** [comment]
I'm unlikely to migrate to vector tiles in this refactoring iteration as all the "task" like objects that is (Notes, Osmose, Maproulette, Vespucci Todos) are displayed in the same layer, but it is definitely possible some time in the future.
